src/transport.h

```diff
@@ -36,10 +47,15 @@ typedef struct {
  *   copied.
  * @param iov_count The number of IOVec structures in the array.
  */
-typedef void (*TXFunction)(Command, uint8_t, const IOVec*,
+typedef bool (*TXFunction)(Command, uint8_t, const IOVec*,
```

Comment:
This seems rather vague for a C typedef. Are you imagining all transmit function pointers will look like this? Or just ones that will use transport?